### ISSUE TITLE
Accessiblity: Header tags on landing page need to start with (h1) and remove one empty (h?) tag.

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/Ottawa-ckan#21]
WCAG 2.0 - Success Criteria - 1.3.1 Level (A) violation 
1.Go to: http://data.ottawa.ca/
2.View source
3.'H1' tag is missing as it starts with 'H2'
4. Also there is an 'empty' heading second last from the bottom of the page.

Resolution: Update tags to begin with 'H1" & remove the empty one (h?) second from the bottom of the page.
![main pg_h list](https://cloud.githubusercontent.com/assets/5271253/8009823/5923ac3e-0b79-11e5-9342-934cc6325e68.JPG)

**Comment by StephenOTT** [comment]
:+1:
